src/components/NavBar/SuggestionRow.tsx

```diff
@@ -22,11 +22,27 @@ import { useCallback, useEffect, useState } from 'react'
 import { Link, useNavigate } from 'react-router-dom'
 import styled from 'styled-components/macro'
 
+import { getDeltaArrow } from '../Tokens/TokenDetails/PriceChart'
 import * as styles from './SearchBar.css'
 
 const StyledLogoContainer = styled(LogoContainer)`
   margin-right: 8px;
 `
+const PriceChangeContainer = styled.div`
+  display: flex;
+  align-items: center;
+`
+
+const PriceChangeText = styled.span<{ delta: number }>`
+  font-size: 14px;
+  line-height: 20px;
+  color: ${({ theme, delta }) => (delta >= 0 ? theme.accentSuccess : theme.accentFailure)};
```

Comment:
nit: would suggest passing in `isNegative` or `isPositive` instead of delta

Reply:
Updated this too